DESCRIPTION

```diff
@@ -9,10 +9,10 @@ Imports:
     desc,
     rex,
     rprojroot,
-    utf8,
+    enc,
```

Comment:
Could you please move this between `desc` and `rex` to maintain alphabetic order?